src/component/project/Project/ProjectFeatureToggles/hooks/useSetFeatureState.ts

```diff
@@ -0,0 +1,31 @@
+import useAPI from 'hooks/api/actions/useApi/useApi';
+import { useCallback } from 'react';
+
+export const useSetFeatureState = () => {
+    const { makeRequest, createRequest, errors } = useAPI({
+        propagateErrors: true,
+    });
+
+    const setFeatureState = useCallback(
```

Comment:
There should already be a hook for this in the useFeaturesApi hook: 

`toggleFeatureEnvironmentOn and toggleFeatureEnvironmentOff`